bin/setup

```diff
@@ -18,23 +16,29 @@ FileUtils.chdir APP_ROOT do
   system! "bin/web 'gem install bundler -v '#{bundler_version}''"
   system('bin/bundle check') || system!('bin/bundle install')
 
-  # puts "\n== Copying sample files =="
+  # puts '\n== Copying sample files =='
   # unless File.exist?('config/database.yml')
   #   FileUtils.cp 'config/database.yml.sample', 'config/database.yml'
   # end
 
-  puts "\n== Preparing database =="
+  puts '\n== Preparing database =='
   system! 'bin/rails db:prepare'
 
-  puts "\n== Removing old logs and tempfiles =="
+  puts '\n== Removing old logs and tempfiles =='
   system! 'bin/rails log:clear tmp:clear'
 
   puts "\n== Restarting application server =="
   system! 'bin/rails restart'
 
-  # puts "\n== Configuring puma-dev =="
+    # puts "\n== Configuring puma-dev =="
   # system "ln -nfs #{APP_ROOT} ~/.puma-dev/#{APP_NAME}"
   # system "curl -Is https://#{APP_NAME}.test/up | head -n 1"
```

Comment:
maybe this should be removed? not sure where it comes from, but I don't see it here https://github.com/rails/rails/blob/v8.0.1/railties/lib/rails/generators/rails/app/templates/bin/setup.tt

Reply:
ahh seems like it used to be there, but no more https://github.com/rails/rails/blob/v7.2.2.1/railties/lib/rails/generators/rails/app/templates/bin/setup.tt